Tests/CarthageKitTests/AlgorithmsSpec.swift

```diff
@@ -2,8 +2,10 @@ import CarthageKit
 import Nimble
 import Quick
 
-class AlgorithmsSpec: QuickSpec {
-	override func spec() {
+// swiftlint:disable:this file_length
```

Comment:
For the tests folder, you mean, right? Cause turning that off entirely really isn't something we should do. There's good reasons to keep files short.